src/TextField.tsx

```diff
@@ -170,6 +174,11 @@ const Text = (props: TextFieldProps) => {
         value={props.value ?? ''}
         {...filterDOMProps(props)}
       />
+{props.helperText &&
+      <HelperText>
+        <HelperTextItem>{props.helperText}</HelperTextItem>
+      </HelperText>}
```

Comment:
Removing this code,, as there is osme magic generating automatically help text as shown bellow. It was enough to add `helperText` to `Props`


```
      <TextField required={false} name="x" helperText="y">
        <Text required={false} name="x" helperText="y" id="uniforms-0000-000f" disabled={false} label="" placeholder="" showInlineError={false} changed={false} error={{...}} errorMessage="" field={{...}} fieldType={[Function: String]} fields={{...}} onChange={[Function (anonymous)]} value={[undefined]}>
          <FormGroup fieldId="uniforms-0000-000f" label="" isRequired={false} validated="default" type={[undefined]} helperText="y" helperTextInvalid="">
            <div type={[undefined]} className="pf-c-form__group">
              <div className="pf-c-form__group-control">
                <TextInput name="x" isDisabled={false} validated="default" onChange={[Function: onChange]} placeholder="" type="text" value="" required={false} helperText="y" id="uniforms-0000-000f">
                  <TextInputBase name="x" isDisabled={false} validated="default" onChange={[Function: onChange]} placeholder="" type="text" value="" required={false} helperText="y" id="uniforms-0000-000f" innerRef={{...}} aria-label={{...}} className="" isRequired={false} isReadOnly={false} isLeftTruncated={false}>
                    <input name="x" placeholder="" required={false} helperText="y" id="uniforms-0000-000f" aria-label={{...}} onFocus={[Function (anonymous)]} onBlur={[Function (anonymous)]} className="pf-c-form-control" onChange={[Function (anonymous)]} type="text" value="" aria-invalid={false} disabled={false} readOnly={false} />
                  </TextInputBase>
                </TextInput>
                <div className="pf-c-form__helper-text" id="uniforms-0000-000f-helper" aria-live="polite">
                  y
                </div>
              </div>
            </div>
          </FormGroup>
        </Text>
      </TextField>

```